python/mathsys/dev/ir/local.py

```
@@ -0,0 +1,53 @@
+    def __new__(self, value: int) -> bytes:
+        if not 1 <= value <= 2**8 - 1: raise ValueError(f"'{value}' is outside range for u8.")
+        return bytes([value])
+
+#> TYPES -> NULL8 CLASS
+class null8:
+    def __new__(self) -> bytes: return bytes([0])
+
+#> TYPES -> U32 CLASS
+class u32:
+    def __new__(self, value: int) -> bytes:
+        if not 1 <= value <= 2**32 - 1: raise ValueError(f"'{value}' is outside range for u32.")
```

Ruff (0.14.6)

Multiple statements on one line (colon)

(E701)